modules/serverless-install-kafka-odc.adoc

```diff
@@ -18,9 +18,13 @@ Cluster administrators can enable the use of Apache Kafka functionality in an {S
 
 . In the *Administrator* perspective, navigate to *Operators* → *Installed Operators*.
 . Check that the *Project* dropdown at the top of the page is set to *Project: knative-eventing*.
-. Click *Knative Kafka* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Kafka* tab.
-. Click *Create Knative Kafka*.
-. Optional: Configure the *KnativeKafka* object in the *Create Knative Kafka* page. To do so, use either the default form provided or edit the YAML.
+. In the list of *Provided APIs* for the {ServerlessOperatorName}, find the *Knative Kafka* box and click *Create Instance*.
+. Configure the *KnativeKafka* object in the *Create Knative Kafka* page.
++
+[IMPORTANT]
+====
+To use the Kafka channel or Kafka source on your cluster, you must toggle the *Enable* switch for the options you want to use to *true*. These switches are set to *false* by default.
```

Comment:
If we point to the need of enabling the channel/source , we should probably also point out the requirement of setting a valid `bootstrapServers` for enabling Kafka channel.

Reply:
Added the Bootstrap Servers requirement to this paragraph.